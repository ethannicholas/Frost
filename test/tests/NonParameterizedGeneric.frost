class NonParameterizedGeneric {
    def error:Collection

    def error2 := Array<Int>()->Array
}