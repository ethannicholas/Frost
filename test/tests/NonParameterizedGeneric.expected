NonParameterizedGeneric.panda:2:15: error: 'panda.collections.Collection' expected 1 generic parameter, but found 0
NonParameterizedGeneric.panda:4:33: error: 'panda.collections.Array' expected 1 generic parameter, but found 0
2 errors
